iib/workers/tasks/build.py

```diff
@@ -496,6 +497,65 @@ def _opm_index_add(
 
     with set_registry_token(overwrite_from_index_token, from_index):
         run_cmd(cmd, {'cwd': base_dir}, exc_msg='Failed to add the bundles to the index image')
+    
+    # add properties to bundles in the index
+    if add_properties:
+        maxOpenShiftVersionProperty = {
+            "type": "olm.maxOpenShiftVersion",
+            "value": "4.8",
+            "operatorbundle_version": "",
+            "operatorbundle-path": "",
+        }
+        for bundle in bundle_str.split(','):
+            if _sdk_is_valid(bundle):
+                _add_property_to_index(base_dir, bundle, maxOpenShiftVersionProperty)
+
+
+@retry(attempts=2, wait_on=IIBError, logger=log)
```

Comment:
There is a config value for this, 2 is overriding that. Good reason to override?